brainscore_vision/models/temporal_model_mae/model.py

```diff
@@ -0,0 +1,66 @@
+from .mae_model import pfMAE
+import torch as th
+
+from brainscore_vision.model_helpers.activations.temporal.model import PytorchWrapper
+from torchvision import transforms
+
+
+class MAEWrapper(PytorchWrapper):
+    def forward(self, inputs):
+        tensor = th.stack(inputs)
+        tensor = tensor.permute(0, 2, 1, 3, 4)
```

Comment:
One question: the mae (and resnet) seems to be a image model, but here there are videos passing in. How does it work?